recipes/mono/mono-windows.recipe

```diff
@@ -39,17 +39,33 @@ class Recipe(recipe.Recipe):
     ]
 
     files_bins = ['mono-sgen']
-    files_bin = ['bin/zlib1.dll',
-                 'bin/intl.dll',
-                 'bin/monosgen-2.0.dll']
+    files_bin = [
+        'bin/al',
```

Comment:
I would move these to deve, since they are only used for development and keep in been just the 3 files we had before.